### PR TITLE
[Julia] Write a Concept Exercise implementation guide

### PR DESCRIPTION
**Pull request by SaschaMann**

### CODE REVIEW

File: languages/julia/docs/implementing-a-concept-exercise.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+            └── $slug
+                ├── .docs
+                |   ├── instructions.md
+                |   ├── introduction.md
+                |   ├── hints.md
+                |   └── after.md (optional)
+                ├── .meta
+                |   └── example.jl
+                ├── $slug.jl
+                ├── $runtests.jl
```
Comment (ErikSchierboom) [inline languages/julia/docs/implementing-a-concept-exercise.md]:
```suggestion
                ├── runtests.jl
```

**Comment by SaschaMann** [comment]
Nice catch, thanks!

**Comment by ErikSchierboom** [comment]
@SaschaMann Do you want another Julia maintainer to look at it, or shall I merge?

**Comment by SaschaMann** [comment]
Thank you both.

**Pull request merged**